test/extended/util/client.go

```diff
@@ -429,11 +430,28 @@ func (c *CLI) TeardownProject() {
 	}
 }
 
+// SetupNamespace creates a new namespace
+func (c *CLI) SetupNamespace() {
+	newNamespace := names.SimpleNameGenerator.GenerateName(fmt.Sprintf("e2e-ns-%s-", c.kubeFramework.BaseName))
+	e2e.Logf("Creating namespace %q", newNamespace)
+	output, errinfo := c.WithoutNamespace().WithoutKubeconf().Run("create").Args("namespace", newNamespace).Output()
+	o.Expect(errinfo).NotTo(o.HaveOccurred())
+	o.Expect(output).Should(o.ContainSubstring("created"))
+	c.currentWs.Namespaces = append(c.currentWs.Namespaces, newNamespace)
+	c.workSpacesToDelete[len(c.workSpacesToDelete)-1].Namespaces = append(c.workSpacesToDelete[len(c.workSpacesToDelete)-1].Namespaces, newNamespace)
+}
+
 // SetupWorkSpace creates a new WorkSpace under the org workspace
 func (c *CLI) SetupWorkSpace() {
 	c.SetupWorkSpaceWithSpecificPath(c.homeServerURL)
 }
 
+// SetupWorkSpaceWithNamespace creates a new WorkSpace under the org workspace and creates a namespace under the WorkSpace
+func (c *CLI) SetupWorkSpaceWithNamespace() {
+	c.SetupWorkSpaceWithSpecificPath(c.homeServerURL)
```

Comment:
Comment is `org workspace` but code `c.homeServerURL` is home workspace

Reply:
Good point! Previously I use the `org workspace`, but after changing of common user couldn't create ws under org ws I forgot update the comment.

Reply:
Addressed.

Reply:
@Phaow added LGTM, but one minor comment: above `// SetupWorkSpace creates a new WorkSpace under the org workspace` is not yet updated.